src/v2/Components/FlashBanner/index.tsx

```diff
@@ -41,3 +70,25 @@ export const FlashBanner: React.FunctionComponent<{
     </Banner>
   )
 }
+
+export const FlashBanner: React.FC = () => {
+  const { relayEnvironment } = useSystemContext()
+
+  // TODO: only wrap in QR if the systemcontext user is present?
+  return (
+    <QueryRenderer
+      environment={relayEnvironment}
+      query={graphql`
+        query FlashBannerQuery {
+          me {
+            canRequestEmailConfirmation
+          }
+        }
+      `}
+      render={({ props, error }) => {
+        if (error) console.error(error)
+        return <BaseFlashBanner {...props} />
```

Comment:
Earlier there was a fragment container here, but now it's just a QueryRenderer passing props into a component because there are no other components needed at this level, so no need to juggle various fragment containers around.